### ISSUE TITLE
Unable to ping any region.

### ISSUE THREAD

**Issue by Cfouria** [ValveSoftware/Dota-2#1559]
#### Your system information

* System information from steam (`Steam` -> `Help` -> `System Information`) in a [gist](https://gist.github.com/): https://gist.github.com/Cfouria/149f525ef5c16d434f49a1393355fc37
* Have you checked for system updates?: [Yes/No] Yes
* Are you using the latest stable video driver available for your system? [Yes/No] Yes
* Have you verified the game files?: [Yes/No] Yes

#### Please describe your issue in as much detail as possible:
Describe what you _expected_ should happen and what _did_ happen. Please link any large pastes as a [Github Gist](https://gist.github.com/).
I opened the matchmaking region selector and all regions, sans China, are "ping: FAILED"
I clicked `refresh pings` expecting improvements and all regions, sans China remain as "ping: FAILED"
China regions are "ping: Unknown" and changes to "ping: Calculating..." briefly when clicking `refresh pings`
Each click of `refresh pings` outputs to the console
```
[SteamNetSockets] Starting ping measurement
[SteamNetSockets] Ping measurement completed
[SteamNetSockets] No ping data available.
```
At the very top of console output is
```
[SteamNetSockets] Failed to communicate with routing cluster 'sgp'.
[SteamNetSockets] Unable to communicate with ANY Steam Datagram routing cluster.  Possible problem with local internet connection?
[SteamNetSockets] Failed to communicate with routing cluster 'sea'.
...
[SteamNetSockets] Failed to communicate with routing cluster 'lax'.
[SteamNetSockets] Ping measurement completed
[SteamNetSockets] No ping data available.
[SteamNetSockets] Failed to communicate with routing cluster 'jnb'.

```
I can select a region and queue .
I cannot join the game that is found.
I can use other tabs (store, watch, learn).
I cannot use the arcade tab. Clicking arcade gives a massive list in console of
`Ignoring custom lobby 25964460769648168, cannot estimate ping to ''`
Arcade tab is mostly empty like it is still loading, except arcade>library
I can arcade>library>game>join chat
I cannot arcade>library>game>play, getting a popup saying
`failed to gather ping information`
and console output of
```
Cannot get ping location for local host
Failed to send QuickJoin request as couldn't get lowest ping server region
```
I can install games to arcade>library
I cannot uninstall games in arcade>library, nothing happens even in console

#### Steps for reproducing this issue:

1. Start Client
2. Open region selector

**Comment by Cfouria** [comment]
further information

- last time played was around august 2017
- the router and computer hardware have not changed
- I practice against bots solo (worked)
- got a new message in console `Net packet timing values don't make sense.  Send time=2592041789761, Recv time=2592041790139, Now=2592041790163, Peer delay=4000`

**Comment by Cfouria** [comment]
Problem is resolved.

It was the router's firewall. Using port forwarding automatically changed other settings. I only checked because `ping google.com` was giving `request timed out`

**Comment by DocJRoberts** [comment]
Cfouria I'm having the exact issue you're describing. Can't ping any servers in the Region select list, can't join a game after accepting resulting in an abandon, can't spectate any games, arcade list is empty. I can only solo bot match. I've tried everything I can think of. What did you end up doing with your router's firewall? I turned mine off entirely and it didn't seem to make a difference :\ Looking for any help I can get at this point
